### PR TITLE
Add unibilium

### PR DESCRIPTION
**Pull request by equalsraf**
Recipe for the unibilium library. The patch disables asan CFLAGS and removes the default terminfo paths.

### CODE REVIEW

**Comment by Alexpux** [comment]
Why we need this package?

**Comment by equalsraf** [comment]
Unibilium is a portable terminfo library so it should be useful for native windows apps that need to deal with a terminal. Personally I'd like to be able to build Neovim using MSYS2 and this is one of its dependencies.

File: mingw-w64-unibilium/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,32 @@
+_realname=unibilium
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=1.2.0
+pkgrel=1
+pkgdesc='A terminfo parsing library'
```
Comment (Alexpux) [inline mingw-w64-unibilium/PKGBUILD]: Add mingw-w64 at the end of description.

File: mingw-w64-unibilium/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+arch=(x86_64 i686)
```
Comment (Alexpux) [inline mingw-w64-unibilium/PKGBUILD]: For mingw packages arch ***MUST*** be always ````any````

**Comment by equalsraf** [comment]
Ok, updated pkgdesc and arch

**Pull request merged**